### ISSUE TITLE
Make labels more meaningful

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#236]
Titles can be improved, wither posed as a question or make the instructional. And we should avoid jargon like "accessibility support" in the titles. In fo in text box explanations should be clarified and use the white space for some instructions. This issue should be worked on together with the Education & Outreach Working Group.
